### PR TITLE
Version 1.2.0

### PR DESCRIPTION
**Pull request by YannickWeber**
**Motivation**

Resolves #<issue>

**Changes**

### CODE REVIEW

File: junit4/src/test/java/com/hivemq/testcontainer/junit4/ContainerWithFileInHomeIT.java [hunk 1]
```diff
@@ -26,35 +26,33 @@
...
-                            .id("extension-1")
-                            .name("my-extension")
-                            .version("1.0")
-                            .mainClass(FileCheckerExtension.class).build())
-                    .withFileInHomeFolder(
-                            new File("src/test/resources/additionalFile.txt"),
-                            "/additionalFiles/")
-                    .withDebugging();
-
     @Test(timeout = 500_000)
```
Comment (SgtSilvio) [inline junit4/src/test/java/com/hivemq/testcontainer/junit4/ContainerWithFileInHomeIT.java:40]: Why are the timeouts so long, over 8 minutes?

File: junit5/src/test/java/com/hivemq/testcontainer/junit5/ContainerWithControlCenter.java [hunk 2]
```diff
@@ -15,30 +15,34 @@
...
 import org.apache.http.client.methods.HttpUriRequest;
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.apache.http.impl.client.HttpClientBuilder;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.Timeout;
-import org.junit.jupiter.api.extension.RegisterExtension;
 
 import java.util.concurrent.TimeUnit;
 
+@SuppressWarnings("ConstantConditions")
```
Comment (SgtSilvio) [inline junit5/src/test/java/com/hivemq/testcontainer/junit5/ContainerWithControlCenter.java:27]: why is this suppression here?

File: junit5/src/test/java/com/hivemq/testcontainer/junit5/ContainerWithControlCenter.java [hunk 3]
```diff
@@ -15,30 +15,34 @@
...
     @Test
     @Timeout(value = 5, unit = TimeUnit.MINUTES)
-    public void testHttp() throws Exception {
+    public void test() throws Exception {
+
+        final HiveMQTestContainerExtension extension =
+                new HiveMQTestContainerExtension("hivemq/hivemq4", "latest")
+                        .withControlCenter(CONTROL_CENTER_PORT);
+
+        extension.beforeEach(null);
```
Comment (SgtSilvio) [inline junit5/src/test/java/com/hivemq/testcontainer/junit5/ContainerWithControlCenter.java:40]: Is it right to pass null here?

File: junit5/src/test/java/com/hivemq/testcontainer/junit5/ContainerWithMavenExtensionIT.java [hunk 4]
```diff
@@ -15,30 +15,33 @@
...
     @Timeout(value = 5, unit = TimeUnit.MINUTES)
-    void test_single_class_extension() throws ExecutionException, InterruptedException {
+    void test() throws Exception {
+        final HiveMQTestContainerExtension extension =
+                new HiveMQTestContainerExtension()
+                        .withExtension(new MavenHiveMQExtensionSupplier("src/test/resources/maven-extension/pom.xml")
+                                .cleanBefore()
+                                .cleanAfter()
+                                .quiet()
+                            .get());
```
Comment (SgtSilvio) [inline junit5/src/test/java/com/hivemq/testcontainer/junit5/ContainerWithMavenExtensionIT.java:40]: weird formatting

File: junit5/src/test/java/com/hivemq/testcontainer/junit5/ContainerWithControlCenter.java (hunk 3 above)
Comment (YannickWeber) [reply junit5/src/test/java/com/hivemq/testcontainer/junit5/ContainerWithControlCenter.java:39]: yes, the method doesn't use the argument anyway.

**Pull request closed without merging**